components/topic-overview/TopicOverviewNav.tsx

```diff
@@ -32,25 +32,34 @@ export const TopicOverviewNav = ({selected, setSelected}:Props) => {
         };
     }, [])
 
+    const scrollToCustomPosition = (id:string) => {
+        setSelected(id)
+        const targetElement = document.getElementById(id);
+        const offset = -50; // Adjust this value to the desired scroll height
```

Comment:
How does this behave in the case of multiple devices? Is the offset consistent across devices?   I would also put this in a constant at the top-level so that it's more descriptive